setup.py

```diff
@@ -8,6 +8,18 @@
 import shutil
 import re
 
+P3 = sys.version_info[0] == 3
```

Comment:
Please change the name of this variable, would rather be very specific rather then calling it P3.